cvl/cvl_when_test.go

```diff
@@ -54,12 +55,11 @@ func TestValidateEditConfig_When_Exp_In_Choice_Negative(t *testing.T) {
 	}
 
 	verifyValidateEditConfig(t, cfgDataRule, CVLErrorInfo{
-		ErrCode:   CVL_SEMANTIC_ERROR,
+		ErrCode:   cvl.CVL_SYNTAX_MISSING_FIELD,
```

Comment:
Please revert this errCode and add the DST_IP to the test payload. Goal of the testcase was to verify the "when" expression which prevents ipv4 configs if IP_TYPE==IPV6. Not sure if there are more test cases to verify such expressions. I think there will be another test case to validate mandatory constraint

Reply:
ok, wasn't sure. The initial test case was just bad.  I'l make that change and see what happens.

Reply:
done